sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -96,7 +111,24 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     programOffering.parentOffering = programOffering;
 
     try {
-      return await this.repo.save(programOffering);
+      const institutionLocation =
```

Comment:
Is this call to get the location data needed since this method receives a `offeringValidationModel` that comes from `buildOfferingValidationModel`?